SPECS/cyrus-sasl/cyrus-sasl.spec

```diff
@@ -2,23 +2,26 @@ Summary:        Cyrus Simple Authentication Service Layer (SASL) library
 Name:           cyrus-sasl
 Version:        2.1.27
 Release:        4%{?dist}
+# License is a BSD with advertising variant 
 License:        Custom
-URL:            https://www.cyrusimap.org/sasl/
-Group:          System Environment/Security
 Vendor:         Microsoft Corporation
 Distribution:   Mariner
+Group:          System Environment/Security
+URL:            https://www.cyrusimap.org/sasl/
 Source0:        https://github.com/cyrusimap/%{name}/releases/download/%{name}-%{version}/%{name}-%{version}.tar.gz
 Patch0:         CVE-2019-19906.patch
-
-BuildRequires:  systemd
-BuildRequires:  openssl-devel
-BuildRequires:  krb5-devel >= 1.12
+# CVE-2020-8032 only applies to the packaging of openSUSE's version of cyrus-sasl
```

Comment:
Could you either provide a link to where that's explained in more detail, or a broader explanation in the .nopatch file itself?